PullRequestCreated/PullRequestCreated.ts

```diff
@@ -42,6 +42,23 @@ class PullRequestCreated extends OctokitAction {
     }
   }
 
+  private additionalFields(parameters: IssueParameters): any {
+    let addutinalFields = {};
+    if (parameters?.parent) {
+      addutinalFields = { parent: { key: parameters.parent } };
+    }
```

Comment:
This looks weird. What about removing `issueType: string` from `createIssue`, make `additionalFields` mandatory and pass issueType via that?
Then here, there would be something like 
```
  return { ...this.parseAdditionalFields(), ...parameters}
```